frontend/src/components/ResourceNameTooltip.tsx

```diff
@@ -1,32 +1,35 @@
 import * as React from 'react';
 import {
+  ClipboardCopy,
   DescriptionList,
   DescriptionListDescription,
   DescriptionListGroup,
   DescriptionListTerm,
-  Icon,
+  Popover,
   Stack,
   StackItem,
-  Tooltip,
 } from '@patternfly/react-core';
 import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
 import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
 import '~/pages/notebookController/NotebookController.scss';
+import DashboardPopupIconButton from '~/concepts/dashboard/DashboardPopupIconButton';
 
 type ResourceNameTooltipProps = {
   resource: K8sResourceCommon;
-  children: React.ReactNode;
+  children?: React.ReactNode;
 };
 
 const ResourceNameTooltip: React.FC<ResourceNameTooltipProps> = ({ children, resource }) => (
   <div>
     {children}{' '}
     {resource.metadata?.name && (
       <div style={{ display: 'inline-block' }}>
-        <Tooltip
+        <Popover
           removeFindDomNode
           position="right"
-          content={
+          showClose={false}
+          withFocusTrap={false}
```

Comment:
Breaking a11y here. cc @DaoDaoNoCode @Gkrumbach07 

We want both of these.

Reply:
Hmm, this brings back that weird issue with the tooltip flashing

Reply:
I didn't see that in my testing -- what browser are you using?

Reply:
Chrome

Reply:
This is very odd -- this issue does not exist on PF docs. Can you log a bug for us to trace down the cause, I think we are misusing the component or we are out of date or there is some weird combo going on here.